### PR TITLE
Web

### PR DESCRIPTION
**Pull request by akotlar**
Various and sundry changes

### CODE REVIEW

File: lib/MPD/Bed.pm [hunk 1]
```diff
@@ -146,6 +146,11 @@ sub _processBedFile {
   my $id = 0;
   for my $line (@lines) {
     my @fields = split /\t/, $line;
+
+    if(@fields == 1) {
+      $self->log( 'fatal', "Bedfile must be tab delimited");
```
Comment (akotlar) [inline lib/MPD/Bed.pm:158]: Someone submitted, several times, a comma delimited file

**Comment by akotlar** [comment]
Added support for bed file to have header. This is the #1 issue that causes jobs to fail currently.

**Comment by akotlar** [comment]
Added better handling of MPD crashes.

**Comment by Jmeigs1** [comment]
Merging and closing.  Similar merge should be done with the master branch.

**Pull request merged**